actionpack/test/controller/redirect_test.rb

```diff
@@ -617,6 +617,16 @@ def test_redirect_to_instrumentation
     assert_equal "http://test.host/redirect/hello_world", payload[:location]
   end
 
+  def test_redirect_with_allowed_redirect_hosts
+    with_raise_on_open_redirects do
+      with_allowed_redirect_hosts(hosts: ["www.rubyonrails.org"]) do
+        get :redirect_to_url
+        assert_response :redirect
+        assert_redirected_to "http://www.rubyonrails.org/"
+      end
+    end
+  end
+
```

Comment:
This also needs a test where the url isn't in the allowed redirect hosts.

Reply:
Thanks, I added that 👍